tic-tac-toe/src/app/components/textInput/index.js

```diff
@@ -1,8 +1,8 @@
 import React, { Fragment } from 'react';
 
-export const CustomInput = props => (
+export const textInput = props => (
```

Comment:
Component names start with upper case

Reply:
Also, use a `default export` for the Component, and destructure the `props` in the arrow function